docs/scripts_md/DBI.md

```diff
@@ -0,0 +1,62 @@
+# NAME
+
+NeuroDB::DBI -- Allows LORIS database connectivity for LORIS-MRI code base
+
+# SYNOPSIS
+
+    use NeuroDB::DBI;
+
+    my $dbh = &NeuroDB::DBI::connect_to_db(@Settings::db);
+
+    my $sth = $dbh->prepare($query);
+    $sth->execute(@bind_value);
+
+    my @row_array  = $sth->fetchrow_array;
+
+# DESCRIPTION
+
+This package performs common tasks relating to database connectivity between
+the LORIS-MRI code base and the LORIS backend database. The following methods
+are available.
```

Comment:
I would remove "the folloeing methods are available" since you have a Methods section right underneath.